include/bmi.hpp

```diff
@@ -0,0 +1,85 @@
+// The Basic Model Interface (BMI) C++ specification.
+//
+// This language specification is derived from the Scientific
+// Interface Definition Language (SIDL) file bmi.sidl located at
+// https://github.com/csdms/bmi.
+
```

Comment:
I would add the version of the BMI this corresponds to as either a macro or a comment (maybe both???) just to be clear.